backend/src/main/java/com/google/sps/data/Time.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import java.util.Calendar;
+
+/** Class to convert Time */
+public final class Time {
+
+  private static Date date;
+
+  /** Constructor */
+  public Time() {
+    date = new Date();
+  }
+
+  /** Constructor with manually set date */
+  public Time(Date date) {
+    this.date = date;
+  }
+
+  /** Return today's date with time of hours:minutes in milliseconds since epoch */
```

Comment:
nit: I think the `/**` makes it a javadoc instead of a comment

Reply:
I meant to have it be a javadoc. Should it not be?

Reply:
Nevermind, it should be. Sorry for the confusion.